themes/digital.gov/layouts/guides/single.html

```diff
@@ -19,7 +19,9 @@
       {{- else -}}
         {{- partial "core/guides/guide-content.html" . -}}
       {{- end -}}
-      {{- partial "core/glossary.html" . -}}
+      {{- if .Parent.Params.glossary -}}
+        {{- partial "core/glossary.html" . -}}
+      {{- end -}}
```

Comment:
Check if `.Parent` page has `glossary` field set. This refers to the root `_index.md` in the guide directory.